mosquitto/Dockerfile

```diff
@@ -2,4 +2,4 @@ FROM eclipse-mosquitto:2.0.10
 
 COPY mosquitto.conf /mosquitto/config/mosquitto.conf
 
-LABEL org.opencontainers.image.source=https://github.com/shankari/everest-demo
+LABEL org.opencontainers.image.source=https://github.com/everest/everest-demo
```

Comment:
This also needs to be `us-joet` for now until we have the image pipeline, and all the related permissions, integrated into everest